dbfit-java/oracle/src/main/java/dbfit/environment/OracleEnvironment.java

```diff
@@ -344,13 +345,24 @@ private DbParameterAccessor makeSingleParam(
             paramDirection = getParameterDirection(direction);
         }
 
-        DbParameterAccessor dbp = new OracleDbParameterAccessor(paramName,
+        DbParameterAccessor dbp = createOracleDbParameterAcccessor(
+                paramName,
                 paramDirection, getSqlType(dataType), getJavaClass(dataType),
                 paramPosition, normaliseTypeName(dataType), userTypeName);
 
         return dbp;
     }
 
+    private OracleDbParameterAccessor createOracleDbParameterAcccessor(
```

Comment:
Ah, we have different list of parameters. Then I think it would be better to keep it as it is (just to make it more obvious that it's a bit different).